src/lib/util.js

```diff
@@ -49,6 +49,17 @@
             return byId( window.location.hash.replace( /^#\/?/, "" ) );
         };
 
+        // `getUrlParamValue` return a given URL parameter value if it exists
+        // `undefined` if it doesn't exist
+        var getUrlParamValue = function( parameter ) {
+            var chunk = window.location.search.split( parameter + "=" )[ 1 ];
```

Comment:
What if there are more than one parameter. For example: `?data-autoplay=5&foo=bar&zoo=zar`

Reply:
The utility is intended to get a single parameter value so for example:
```javascript
window.location = '...?data-autoplay=5&foo=bar&zoo=zar'

getUrlParamValue('zoo') === 'zar'
getUrlParamValue('foo') === 'bar'
```

I did it that way because my use case is kind of limited to a single value. Do we need a utility to parse multiple URL parameters ?

Reply:
Sorry, my bad. I didn't read properly. The next line takes care of splitting.

(I would have split by & first, so I was confused. But your approach works too, I now see.)

Reply:
Awesome, no worries at all!